app/feedpeople/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class FeedPerson(models.Model):
+  eppn = models.CharField(max_length=255, blank=True, null=True)
+  firstname = models.CharField(max_length=255, blank=True, null=True)
+  lastname = models.CharField(max_length=255, blank=True, null=True)
+  location = models.CharField(max_length=255, blank=True, null=True)
+  name = models.CharField(max_length=255, blank=True, null=True)
+
+  def __str__(self):
+    return f"{self.firstname} {self.lastname}"
```

Comment:
I realize that I included this in the spec, so apologies for that, but let's change the `__str__` method to use the `name` field, rather than a concatenation of `firstname` and `lastname`. This way we can use the same field for __str__ on all of the models.

Reply:
No worries, thanks for clarifying! The return value of `__str__` was updated to use the `name` field instead in bc6c77b.